sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonSideComponentsElement.java

```diff
@@ -81,36 +92,62 @@ default void addSideComponents(
 		Consumer<Runnable> showUnsavedChangesPopup) {
 
 		UserProvider currentUser = UserProvider.getCurrent();
+		CaseListComponent caseListComponent = null;
+		ContactListComponent contactListComponent = null;
+		EventParticipantListComponent eventParticipantListComponent = null;
 
 		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.CASE_SURVEILANCE)
 			&& currentUser != null
 			&& currentUser.hasUserRight(UserRight.CASE_VIEW)) {
-			layout.addComponent(
-				new SideComponentLayout(
-					new CaseListComponent(person, entityType == CoreEntityType.CASE ? entityUuid : null, showUnsavedChangesPopup)),
-				CASES_LOC);
+			caseListComponent = new CaseListComponent(person, entityType == CoreEntityType.CASE ? entityUuid : null, showUnsavedChangesPopup);
+			layout.addComponent(new SideComponentLayout(caseListComponent), CASES_LOC);
 		}
 
 		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.CONTACT_TRACING)
 			&& currentUser != null
 			&& currentUser.hasUserRight(UserRight.CONTACT_VIEW)) {
-			layout.addComponent(
-				new SideComponentLayout(
-					new ContactListComponent(person, entityType == CoreEntityType.CONTACT ? entityUuid : null, showUnsavedChangesPopup)),
-				CONTACTS_LOC);
+			contactListComponent =
+				new ContactListComponent(person, entityType == CoreEntityType.CONTACT ? entityUuid : null, showUnsavedChangesPopup);
+			layout.addComponent(new SideComponentLayout(contactListComponent), CONTACTS_LOC);
 		}
 
 		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.EVENT_SURVEILLANCE)
 			&& currentUser != null
 			&& currentUser.hasUserRight(UserRight.EVENT_VIEW)
 			&& currentUser.hasUserRight(UserRight.EVENTPARTICIPANT_VIEW)) {
-			layout.addComponent(
-				new SideComponentLayout(
-					new EventParticipantListComponent(
-						person,
-						entityType == CoreEntityType.EVENT_PARTICIPANT ? entityUuid : null,
-						showUnsavedChangesPopup)),
-				EVENT_PARTICIPANTS_LOC);
+			eventParticipantListComponent = new EventParticipantListComponent(
+				person,
+				entityType == CoreEntityType.EVENT_PARTICIPANT ? entityUuid : null,
+				showUnsavedChangesPopup);
+			layout.addComponent(new SideComponentLayout(eventParticipantListComponent), EVENT_PARTICIPANTS_LOC);
+		}
+
+		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.SAMPLES_LAB)
+			&& currentUser != null
+			&& currentUser.hasUserRight(UserRight.SAMPLE_VIEW)
+			&& getClass().equals(PersonDataView.class)) {
+
+			List<String> casePersonList = caseListComponent == null
```

Comment:
the variable names should not contain *Person* unless you don't store person uuids

Reply:
renamed